pkg/manager/daemon_adaptor.go

```diff
@@ -192,6 +199,25 @@ func (m *Manager) BuildDaemonCommand(d *daemon.Daemon, bin string, upgrade bool)
 
 	cmd := exec.Command(nydusdPath, args...)
 
+	if config.IsKeyringEnabled() && !d.IsSharedDaemon() {
```

Comment:
Can we support shared daemon?

Reply:
Yes, it call `c.FillAuth(keyChain)` to fill auth before sending mount request to nydus daemon.